core/src/empty/empty.module.css

```diff
@@ -0,0 +1,14 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: Inter;
```

Comment:
Font family should be used in css variables @thien-do

Reply:
@ldakhoa is correct, but we should not even define the font-family here. It should be inherited anyway.

Also declaring a font like this is quite bad. There should be fallback, like at least "Inter, sans-serif"